sparkpost/django/email_backend.py

```diff
@@ -55,7 +57,7 @@ def _send(self, message):
                         'Content type %s is not supported' % alternative[1]
                     )
 
-        return self.client.transmissions.send(**params)
+        return self.client.transmissions.send(**params, **options)
```

Comment:
Maybe you can't, but I can, because I'm using Python 3.5 :sunglasses: 

But yeah, I'll fix it so that the older Pythons are supported too.